rpi-setup.sh

```diff
@@ -109,6 +109,10 @@ cat > $vesselJson <<jsonfile
 }
 jsonfile
 
+sudo chmod 644 $vesselJson
+sudo chown pi $vesselJson
```

Comment:
$SUDO_USER does the trick, it finds the user who invoked sudo. Not sure about group though